pywb/apps/frontendapp.py

```diff
@@ -289,25 +289,22 @@ def serve_static(self, environ, coll='', filepath=''):
         except Exception:
             self.raise_not_found(environ, 'static_file_not_found', filepath)
 
-    def get_metadata(self, coll):
-        """Retrieve the metadata associated with a collection
+    def get_coll_config(self, coll):
+        """Retrieve the collection config, including metadata, associated with a collection
 
-        :param str coll: The name of the collection to receive metadata for
-        :return: The collections metadata if it exists
+        :param str coll: The name of the collection to receive config info for
+        :return: The collections config. The 'metadata' dict contains optional user metadata
```

Comment:
```suggestion
        :return: The collections config
```

I think that `The 'metadata' dict contains optional user metadata` would work better in the description of the method not the return value. 

At least to me it reads as if a tuple of two dictionaries was being returned